### ISSUE TITLE
Error "Nickname in use" does not use the alternative nickname after reconnecting

### ISSUE THREAD

**Issue by Zarthus** [thelounge/thelounge#3017]
### Feature Description

Currently if your nickname is in use after a reconnect, thelounge does nothing.

The following would be sent until the registration times out:
```
21:43:57  Network created, connecting to irc.quakenet.org:6667...
21:43:57  Connected to the network.
21:43:57 <irc.quakenet.org>  *** Looking up your hostname
21:43:57 <irc.quakenet.org>  *** Checking Ident
21:43:57 <irc.quakenet.org>  *** Found your hostname
21:43:58 Zarthus: Nickname is already in use.
21:43:58 <irc.quakenet.org>  *** Got ident response
21:43:58  You're now known as Zarthus5
21:43:58 [002] Your host is cymru.us.quakenet.org, running version u2.10.12.10+snircd(1.3.4a)
21:43:58 [003] This server was created Mon Nov 6 2017 at 17:26:27 UTC
21:43:58 [004] cymru.us.quakenet.org u2.10.12.10+snircd(1.3.4a) dioswkgxRXInP biklmnopstvrDcCNuMT bklov
21:43:58 [251] There are 16 users and 15166 invisible on 29 servers
21:43:58 [252] 56 operator(s) online
21:43:58 [254] 12909 channels formed
21:43:58 [255] I have 900 clients and 1 servers
21:43:58 <cymru.us.quakenet.org>  Highest connection count: 2414 (2413 clients)
21:43:58   MOTD File is missing
21:43:58 <cymru.us.quakenet.org>  on 4 ca 1(4) ft 20(20) tr
21:43:58  * Zarthus5 sets mode +i
21:44:13  Disconnected from the network, and will not reconnect. Use /connect to reconnect again.
21:44:15  Connected to the network.
21:44:15 <irc.quakenet.org>  *** Looking up your hostname
21:44:15 <irc.quakenet.org>  *** Checking Ident
21:44:15 <irc.quakenet.org>  *** Found your hostname
21:44:15 Zarthus: Nickname is already in use.
21:44:15 <irc.quakenet.org>  *** Got ident response
21:44:45 [451] * Register first.
```

This specifically happens on secondary connects; upon network creation all is fine

Suggestion: Handle numeric 451 by adding `"/nick "` to the input area and/or displaying the user a message a new nick needs to be supplied.

**Comment by xPaw** [comment]
How do you expect the UI to work if you have the client closed or open in multiple browsers?

**Comment by xPaw** [comment]
With the renamed issue, it should be fixed by https://github.com/kiwiirc/irc-framework/pull/189

**Comment by xPaw** [comment]
Fixed by #3040
